src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/IntegrationTestSupportController.java

```diff
@@ -57,9 +64,24 @@ public Claim updateRespondByDate(
         return getClaim(claimReferenceNumber, authorisation);
     }
 
+    @PutMapping("/claims/{claimReferenceNumber}/defendant/{defendantUsername}")
+    public Claim linkDefendantToClaim(
+        @PathVariable("claimReferenceNumber") String claimReferenceNumber,
+        @PathVariable("defendantUsername") String defendantUsername,
+        @RequestHeader(value = HttpHeaders.AUTHORIZATION, required = false) String authorisation
+    ) {
+        Claim claim = getClaim(claimReferenceNumber, authorisation);
+
+        //todo how we should store default password? const? properties?
+        User defendant = this.userService.authenticateUser(defendantUsername, "Password12");
```

Comment:
@dharmendrak would it be possible to update the case as anonymous caseworker?

Reply:
@dharmendrak ☝️ 

The story was created on the assumption that anonymous caseworker will perform the operation and there won't be user JWT required.